src/moonstone-samples/src/ContextualPopupSample.js

```diff
@@ -233,8 +231,14 @@ module.exports = kind({
 		{from: '.$.nestedRadioGroup.active.content', to: '.$.nestedRadioValue.content', transform: function(val){
 			this.dismissRadioSelection();
 			return val;
-		}}
+		}},
+		{from: '$.btnSpotlightModal.value', to: '$.buttonPopup.spotlightModal', oneWay: false},
+		{from: '$.btnSpotlightModal.value', to: '$.buttonPopup.autoDismiss', kind: InvertBooleanBinding, oneWay: false},
+		{from: '$.btnModal.value', to: '$.buttonPopup.modal', oneWay: false}
 	],
+	modelToggled: function(inSender, inEvent) {
```

Comment:
This handler is no longer needed, right? It should also have probably been `modalToggled`, originally. :smile: